tests/data/install/pipenv_file/setup

```diff
@@ -0,0 +1,15 @@
+"""A setup.py file for requirements.txt in editable mode."""
```

Comment:
Is this file needed? It feels like only Pipfile.lock is required.

Reply:
It's not needed. I've used `tests/data/install/pipenv_editable` folder as a bootstrap for my test case fixtures and didn't think twice about it. Removing it now.